src/ripple/server/impl/BaseWSPeer.h

```diff
@@ -169,6 +169,7 @@ BaseWSPeer(
         io_service, journal)
     , request_(std::move(request))
     , timer_(io_service)
+    , payload_ ("12345678")
```

Comment:
Might be useful to add a comment mentioning this value is meaningless and that we only care about establishing a size. You might also consider this text  `\,,/(^_^)\,,/`  :)